### DISCUSSION TITLE
Hubs Cloud AWS | Backup and Restore

### DISCUSSION THREAD

**Discussion by Vhive2012** [mozilla/hubs#5256] [Hubs Cloud Questions] [answered]
Good Day,

I was trying to upgrade my stack template within Cloud Formation and followed the below.

https://hubs.mozilla.com/docs/hubs-cloud-aws-updating-the-stack.html

I was trying to take it offline but kept getting the attached error.

I've done an upgrade before which went fine but this one didn't want to work and the stack got stuck on deleting/rolling back from when it failed so I had to delete the stack.

We do have backups in place on AWS and I followed the below but and it seems it did restore but when I try go to the domain [vhivetest.com](http://vhivetest.com/) it keeps giving me a 502 Bad Gateway Error.

Within EC2 under target groups it shows that the target group for the above mentioned stack URL is unhealthy so not sure what is causing that issue or if it's an issue with the restore?

Please can someone assist me?

Has anyone else had this issue and found a fix?

Thanks.
![WhatsApp Image 2022-03-16 at 2 05 03 PM](https://user-images.githubusercontent.com/54393507/158818899-17ddef6b-629d-497d-9ba0-4fe45d81343e.jpeg)

**Comment by agriggs2** [comment, marked as answer]
Hey @Vhive2012, 

You can manually delete the record, I'd take a back up of it before deleting (copy and paste into a txt file) in case you need to restore the dns record, then try restoring your stack. 

FYI: discord is a good place to get this kind of help in real-time.
